server/src/main/java/org/elasticsearch/monitor/jvm/HotThreads.java

```diff
@@ -41,6 +42,19 @@
     private ReportType type = ReportType.CPU;
     private boolean ignoreIdleThreads = true;
 
+    private static final List<String[]> knownElasticIdleFrames = Arrays.asList(
+        new String[] {"java.util.concurrent.ThreadPoolExecutor", "getTask"},
+        new String[] {"sun.nio.ch.SelectorImpl", "select"},
+        new String[] {"org.elasticsearch.threadpool.ThreadPool$CachedTimeThread", "run"},
+        new String[] {"org.elasticsearch.indices.ttl.IndicesTTLService$Notifier", "await"},
+        new String[] {"java.util.concurrent.LinkedTransferQueue", "poll"}
+    );
```

Comment:
These are clearly a mixture of JDK and elastic frames, but it's probably ok to call them `ElasticIdle` ?

Reply:
Good point, I renamed the variable to simply say idle.